### ISSUE TITLE
locality attribute error message in 8.17rc1

### ISSUE THREAD

**Issue by andrew-appel** [coq/coq#17353]
In Coq 8.17rc1, an Instance without a locality attribute gives me an error, not a warning; but the error message still says "scheduled to change in future releases".  Time to change the error message?

```
The default value for instance locality is currently "local"
in a section and "global" otherwise, but is scheduled to
change in a future release. For the time being, adding
instances outside of sections without specifying an explicit
locality attribute is therefore deprecated. It is recommended
to use "export" whenever possible. Use the attributes
#[local], #[global] and #[export] depending on your choice.
For example: "#[export] Instance Foo : Bar := baz."
[deprecated-instance-without-locality,deprecated]
```

**Comment by SkySkimmer** [comment]
Duplicate https://github.com/coq/coq/issues/17309

**Comment by Zimmi48** [comment]
Thanks for the report anyway. The message will be fixed in 8.17.0. It is still time to comment on the proposal currently at https://github.com/coq/coq/pull/17343.
